package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "remark-slate",
-  "version": "1.8.6",
```

Comment:
Can you unstage this change? I'll make the release after this is merged

Reply:
Same with the lock file diff

Reply:
All set! Apologies, work habit committing the lock file :)